ci/default.rb

```diff
@@ -40,11 +40,24 @@
     task before_script: ['ci:common:before_script']
 
     task :lint do
-      sh %(find . -name '*.py' -not \\( -path '*.cache*' -or -path '*embedded*' -or -path '*venv*' \\) | xargs -n 1 pylint --rcfile=./.pylintrc)
+      output = %x[git status --porcelain]
+      paths = output.lines.collect{ |p|
+        p.strip.split ' '
+      }.select{ |stat_path|
+        status = stat_path.first
+        path = stat_path.last
+        path.end_with? ".py" and status != 'D'
```

Comment:
I'm not sure to understand: if `git status --porcelain` returns nothing (or there is no `*.py` files, the old `rake:lint` task will run (`if paths.empty?`). (see for instance on Travis)
I really wanted to have this behavior to have a fast `rake ci:run` when working on something (ie right now, it takes ~15s when a `.py` file is changed, versus more than 2min because of pylint (which is is really slooooow)).